emission_events/emission_events/settings/local.py

```diff
@@ -24,9 +24,14 @@
 
 # https://docs.djangoproject.com/en/1.7/ref/settings/#databases
 DATABASES = {
-    'default': dj_database_url.config(
-        default=os.environ.get('EMISSIONS_DATABASE')
```

Comment:
I didn't intend to push any changes to local.py. Will revert

Sent from my iPhone

> On Nov 3, 2015, at 6:18 PM, Ryan Murphy <notifications@github.com> wrote:
> 
> In emission_events/emission_events/settings/local.py:
> 
> > @@ -24,9 +24,14 @@
> >  
> >  # https://docs.djangoproject.com/en/1.7/ref/settings/#databases
> >  DATABASES = {
> > -    'default': dj_database_url.config(
> > -        default=os.environ.get('EMISSIONS_DATABASE')
> The current scraper probably depends on this, so we probably shouldn't just rip it out.
> 
> —
> Reply to this email directly or view it on GitHub.
>